docs/semgrep-app/integrations.md

```diff
@@ -1,19 +1,163 @@
+
+#### See also
+[Notifcations -> Email](notifications.md/#email)
+
+### Jira
+
+Jira integration is a feature available in Semgrep's Team tier and above.
+
+This integration allows you to create Jira tickets directly from the **Findings** page with relevant information about a particular finding.
+
+To set up Jira integration:
+
+1. In **Integrations,** click **Add Integration**.
+2. Click on **Jira.**
+3. Enter a **Name** for the integration.
+4. Enter the **email address** used for the Atlassian account.
+5. Enter your Atlassian **domain URL**.
+6. Enter your **Project key**. This is the prefix for tasks created within a project. Semgrep will create issues to the project identified here.
+7. Enter the **Issue type.** This is the type of issue for Semgrep findings, for example, *Bug.*
+8. Enter the **API Token**. Tokens are generated through this link: [Manage API Tokens](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account).
+9. Click **Save.**
+
+To create a Jira ticket from Semgrep:
+
+1. In **Findings**, click on the **three-dot icon** of the entry to create a Jira ticket for the finding.
+![Creating a Jira ticket from the Findings page](../img/jira-findings-page.png)<br />
+2. Select **Create issue with Semgrep Jira Integration.**
 ![Output of Jira integration](../img/jira-template.png)
 
+
+### Webhooks
+
+Webhooks are a feature available in Semgrep's Team tier and above.
+
+Webhooks are a generic method for Semgrep to post JSON-formatted findings after each scan to your URL endpoint.  To set up a webhook:
+
+1. In **Integrations,** click **Add Integration.**
+2. Click **Webhook.**
+3. Enter a **Name** for the integration.
+4. Enter the **Webhook URL.**
+5. Select the **Inventory** check box if you would like to receive notifications about Code Asset Inventory findings.
+6. To ensure that Semgrep can post to your URL, click **Test.** 
+![Successful webhook integration test](../img/webhook-successful-test.png)<br />
+7. Click **Save.**
+8. Remember to turn notifications on by going to the **Rule board**, clicking on the **gear icon,** then clicking on the integration's toggle.
+
+Here is a sample of a webhook sent from Semgrep with findings:
+
+![Screenshot of Semgrep webhook JSON with findings ](../img/integrations-webhook-findings.png)<br />
+
+
+#### See also:
+
+[Notifcations -> Webhooks](notifications.md/#webhooks)
+
+### Amazon S3
+
+1. In **Integrations,** click **Add Integration.**
+2. Click **AWS S3.**
+3. Enter the **Channel name** for the integration.
+4. Select the **Inventory** check box if you would like to receive notifications about Code Asset Inventory findings.
+5. To ensure that Semgrep can post to your channel, click **Test.** 
+6. Click **Save.**
+7. Remember to turn notifications on by going to the **Rule board**, clicking on the **gear icon,** then clicking on the integration's toggle.
```

Comment:
```suggestion
7. To enable notifications, go to the **Rule board**, click on the **gear icon**, and then click on the integration's toggle.
```

Is it the integration's toggle or integrations toggle? Should this word be bold as well?
Use imperative as much as possible. It is quicker to read and readers understand what they need to do quicker. Again, please, do not use bold tags or any other tags for the characters which are not part of the words you want to emphasize.